lib/event_bg_bus.rb

```diff
@@ -31,6 +33,20 @@ def publish(event_name, payload = {})
     alias :announce :publish
     alias :broadcast :publish
 
+    private
+
+    def validEvent?(event_env_var, event_name)
+      ENV.include?(event_env_var) && ENV[event_env_var].split(',').include?(event_name)
```

Comment:
I wonder about `starts_with?` instead of include - might be easier to roll with some prefixes